java/RSPEC-4274.java

```diff
@@ -0,0 +1,5 @@
+class PublicAssertParam {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._